static/js/publisher/release/components/releasesConfirmDetails/globalRow.js

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import { InteractiveProgressiveBar } from "../progressiveBar";
+
+const GlobalRow = ({
+  useGlobal,
+  toggleGlobal,
+  globalPercentage,
+  updatePercentage
+}) => {
+  return (
+    <div className="p-releases-confirm__details-global row u-vertically-center">
+      <div className="col-4">
+        <label>
+          <span>
```

Comment:
Is this additional span needed around the one with tooltip?

Reply:
This one is not, removing